### DISCUSSION TITLE
Unable to flash planck with extended thumbs option

### DISCUSSION THREAD

**Discussion by iitsalex** [manna-harbour/miryoku#235] [Q&A] [answered]
Following the extended thumbs option below, the command doesn't work: 
`make planck/rev6:manna-harbour_miryoku:flash FORCE_LAYOUT=ortho_4x12 MIRYOKU_MAPPING=EXTENDED_THUMBS`

https://github.com/manna-harbour/miryoku_qmk/tree/miryoku/users/manna-harbour_miryoku#ortho_4x12

The keyboard remains with the stock layout which has the thumb keys aligned with the inner columns. I was able to build a QWERTY layout version of miryoku with the qmk flash and make commands, but extended thumbs refuses to work.

**Comment by iitsalex** [comment]
I should add, the firmware compiles and flashes fine, but the layout is incorrect (default instead of extended thumbs).

**Comment by manna-harbour** [comment]
What you have looks correct. It's possible there's been a change in QMK relating to `FORCE_LAYOUT` or to the Planck keyboard definition. I'll look into it.

If you just want to get it working for now you could force it in other ways, e.g. remove https://github.com/qmk/qmk_firmware/tree/master/layouts/community/planck_mit/manna-harbour_miryoku or https://github.com/qmk/qmk_firmware/blob/3f192722b1629e4a4b6fe171d7efc21c3a0d3187/keyboards/planck/rev6/info.json#L35.

You could also try putting something bogus here https://github.com/qmk/qmk_firmware/blob/3f192722b1629e4a4b6fe171d7efc21c3a0d3187/layouts/community/ortho_4x12/manna-harbour_miryoku/config.h#L32 to force an error, just to confirm that the correct mapping is being selected.

Also, here's a workflow build you could try flashing just to narrow things down https://github.com/manna-harbour/miryoku_qmk/actions/runs/5913759087.

**Reply by iitsalex** [reply]
Just tried the workflow build and it's also not working as intended from what I can see. Could be a change in QMK like you mentioned. Testing out the other methods shortly.

**Comment by manna-harbour** [comment, marked as answer]
From our discord discussion, it seems to be an issue with version of QMK in the miryoku branch, and isn't present in QMK master.
